File: styler-service/src/main/java/com/lookgen/styler/domain/Photo.java
package com.lookgen.styler.domain;

import jakarta.persistence.*;

@Entity
@Table(name = "photo")
public class Photo {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id = 0L;

    @Column(name = "user_id")
    private Long userId = 0L;

    private String url = "";

    private short ord = 0;

    public Photo() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public short getOrd() {
        return ord;
    }

    public void setOrd(short ord) {
        this.ord = ord;
    }
}

File: styler-service/src/main/kotlin/com/lookgen/styler/domain/Photo.kt
package com.lookgen.styler.domain

import jakarta.persistence.*

@Entity
@Table(name = "photo")
class Photo(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0,

    @Column(name = "user_id")
    var userId: Long = 0,

    var url: String = "",

    var ord: Short = 0
)
